libcontainer/console_linux.go

```diff
@@ -75,21 +58,17 @@ func (c *linuxConsole) Close() error {
 
 // mount initializes the console inside the rootfs mounting with the specified mount label
 // and applying the correct ownership of the console.
-func (c *linuxConsole) mount(rootfs, mountLabel string) error {
+func (c *linuxConsole) mount(mountLabel string) error {
 	oldMask := syscall.Umask(0000)
 	defer syscall.Umask(oldMask)
-	if err := label.SetFileLabel(c.slavePath, mountLabel); err != nil {
```

Comment:
Ask @mrunalp, it's his patch. From my understanding it's because the console is already tagged by SELinux if you open it from inside the container (if you open it from outside you have to tag it manually which is what the code used to do)?